pkgs/applications/networking/pyload/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, fetchpatch, python, gocr, unrar, rhino, spidermonkey }:
+let
+  pythonEnv = (python.withPackages (ps: with ps; [
+    pycurl jinja2 beaker thrift simplejson
+    #optional packages
+    #TODO add Send2Trash
+    #TODO package the other optional ones, see homepage
+    pycrypto feedparser pyqt4
+  ]));
+in
+stdenv.mkDerivation rec {
+  version = "stable";
+  name = "pyLoad-" + version;
+
+  src = fetchFromGitHub {
+    owner = "pyload";
+    repo = "pyload";
+    rev = "stable";
```

Comment:
This has to be a fixed revision, for example a tag / release. In the case of a tag / release, you can also use the tarball.